app/services/chat_channels/update.rb

```diff
@@ -0,0 +1,15 @@
+module ChatChannels
```

Comment:
I don't think we need this at all as it's simply replaceable with `channel.update(params)`